.github/workflows/build.yml

```diff
@@ -100,9 +94,9 @@ jobs:
                 arm: [macOS, ARM64]
                 intel: [macos-latest]
             python:
-              major-dot-minor: '3.7'
-              cibw-build: 'cp37-*'
-              matrix: '3.7'
+              major-dot-minor: '3.10'
+              cibw-build: 'cp310-*'
+              matrix: '3.10'
```

Comment:
This section is an exclude, so can just be deleted - otherwise will have the effect of excluding 3.10 for macos arm (we previously had to exclude 3.7 on mac arm because it didn't exist, but 3.7 was otherwise in the defaults).

Reply:
Friday before holiday read too fast...

Reply:
Going to want to delete this whole exclude list item - the following was the specific combination that was excluded, that is no longer needed (just couldn't leave the PR comment on this whole block since not all of it was changed)

```          
          - os:
              name: macOS
              matrix: macos
              runs-on:
                arm: [macOS, ARM64]
                intel: [macos-latest]
            python:
              major-dot-minor: '3.7'
              cibw-build: 'cp37-*'
              matrix: '3.7'
            arch:
              name: ARM
              matrix: arm
```

All that can go away